routers/user/oauth.go

```diff
@@ -10,16 +10,14 @@ import (
 	"net/url"
 	"strings"
 
-	"github.com/dgrijalva/jwt-go"
-	"github.com/go-macaron/binding"
-
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/auth"
 	"code.gitea.io/gitea/modules/base"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/util"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
blank line

Reply:
But it is used in L124 (and various other places)

Reply:
@kolaente all `code.gitea.io` packages should be splitted with external packages with blank line. See https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
Oh, now I understand what you mean, I though you wanted me to import it as `_`.

Reply:
Fixed.